.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: RELEASE
+# Triggers the workflow on push  to the release branch
+on:
+  push:
+    branches: [ release ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
can we use centos?

Reply:
looks like centos not available https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources